percy/snapshot.py

```diff
@@ -64,18 +114,24 @@ def percy_snapshot(driver, name, **kwargs):
       "For more information on usage of PercyScreenshot, "\
       "refer https://www.browserstack.com/docs/percy/integrate/functional-and-visual")
 
-
     try:
         # Inject the DOM serialization script
         driver.execute_script(fetch_percy_dom())
+        cookies = driver.get_cookies()
```

Comment:
This will be a list of dictionaries and browser expects it in different format we need to add them in proper format.

Reply:
This is also correct format here it return { 'name': '', 'value': ''} in this format and our cli works well with it.

Reply:
Thats not the current format of the cookies in CLI though. once that are coming from document.cookies

Reply:
Since [this](https://github.com/percy/cli/blob/dffcee04e5aefc3cc7f9f84ceaa655ad5adf9972/packages/core/src/discovery.js#L199) check will not become true so cookies are in correct format only as we need to pass in cdp